build-parser.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
+
```

Comment:
I suggest to set some bash flags as explained here to make this script exit more early in case of problems: https://sipb.mit.edu/doc/safe-shell/